Source/Core/VideoCommon/Statistics.cpp

```diff
@@ -183,8 +183,8 @@ void Statistics::DisplayScissor()
     ImGui::EndDisabled();
     ImGui::Checkbox("Show Viewports", &show_viewports);
     ImGui::Checkbox("Show Text", &show_text);
-    ImGui::DragInt("Scale", &scissor_scale, .2f, 1, 16);
-    ImGui::DragInt("Expected Scissor Count", &scissor_expected_count, .2f, 0, 16);
+    ImGui::DragInt("Scale", (int*)&scissor_scale, .2f, 1, 16);
```

Comment:
This is technically invoking undefined behavior. I'd leave these as ints and cast if other types are needed elsewhere.